tests/schema/tests.py

```diff
@@ -554,7 +554,7 @@ def test_add_field_temp_default(self):
         new_field.set_attributes_from_name("surname")
         with connection.schema_editor() as editor:
             editor.add_field(Author, new_field)
-        # Ensure the field is right afterwards
+        # Ensure the field is right afterward
```

Comment:
THB, these comments don't have any value, I'd remove them instead.

Reply:
Yes. I agree, but thought it may be out of scope here.

Shall I add as a separate clean-up to be merged first?

Reply:
We can do this in a single commit. Maybe with a sub-message:
```
This also removes useless comments with the previous spelling.
```